### ISSUE TITLE
Write function to check for wepp executable

### ISSUE THREAD

**Issue by jarad** [jarad/WEPPR#30]
Some functions, e.g. run_wepp, require the wepp executable. We should create a function that does the check for us, e.g. 

    path <- system("which WEPP 2>/dev/null || echo FALSE")

**Comment by jarad** [comment]
[Relevant discussion](https://community.rstudio.com/t/best-practice-to-check-if-an-executable-is-installed/33775)

**Comment by jarad** [comment]
Looks like the base R function `Sys.which()` should be helpful.

**Comment by jarad** [comment]
Verified `Sys.which()` appears to work well. 

I [followed these instructions](https://askubuntu.com/questions/322772/how-do-i-add-an-executable-to-my-search-path) to add the wepp executable to `~/bin` and then add this directory to the path using my `.bashrc` file. 

Then, I started R and got the following

    > Sys.which("wepp")
                      wepp
    "/home/niemi/bin/wepp"

If wepp is not found, then it would return an empty string.
